docs/client_tools/plugins/deadline.md

```diff
@@ -0,0 +1,124 @@
+
+Just as with any other instance in a Deadline managed farm, Conductor instances run the Deadline worker.
+
+*As of Deadline 10.1, the term **Slave** has been replaced with **Worker***
+
+## Installation
+
+![](../../image/download_icon.png) Download the *[plugin](https://github.com/ConductorTechnologies/deadline_plugin)* from Github
+
+*Steps 3-5 require Super User access in Deadline Monitor*
+
+1. Start *[Conductor uploader and downloader daemons]()*
+2. Copy plugins to the correct folders
+    1. `cp deadline_plugin/custom/scripts/Jobs/* -> <your_deadline_repo>/custom/scripts/Jobs/`
+    2. `cp deadline_plugin/custom/events/ConductorWorker -> <your_deadline_repo>/custom/events/`
+    3. `cp deadline_plugin/custom/scripts/General/shutdown_conductor_instance.py -> your_deadline_repo>custom/scripts/General/`
+
+3. Add the path of the conductor client tools to the python paths in your repo
+![](../../image/deadline/python_path_repo.jpg) 
+4. Add the Conductor paths to the MayaCmd plugin. You only need to add for versions that you intend to use
+![](../../image/deadline/maya_path_repo.jpg)
+> - `/opt/autodesk/maya-io/2016.5/maya-io2016.5.SP0/bin/Render`
+> - `/opt/autodesk/maya-io/2016/maya-io2016.SP0/bin/Render`
+> - `/opt/autodesk/maya-io/2017/maya-io2017.SP0/bin/Render`
+> - `/opt/autodesk/maya-io/2017/maya-io2017.SP3/bin/Render`
+> - `/opt/autodesk/maya-io/2017/maya-io2017.SP4/bin/Render`
+> - `/opt/autodesk/maya-io/2017/maya-io2017.SP5/bin/Render`
+> - `/opt/autodesk/maya-io/2018/maya-io2018.SP0/bin/Render`
+> - `/opt/autodesk/maya-io/2018/maya-io2018.SP1/bin/Render`
+> - `/opt/autodesk/maya-io/2018/maya-io2018.SP2/bin/Render`
+> - `/opt/autodesk/maya-io/2018/maya-io2018.SP3/bin/Render`
+> - `/opt/autodesk/maya-io/2018/maya-io2018.SP4/bin/Render`
+> - `/opt/autodesk/maya-io/2018/maya-io2018.SP5/bin/Render`
+> - `/opt/autodesk/maya-io/2018/maya-io2018.SP6/bin/Render`
+> - `/opt/autodesk/maya-io/2019/maya-io2019.SP0/bin/Render`
+> - `/opt/autodesk/maya-io/2019/maya-io2019.SP1/bin/Render`
+> - `/opt/autodesk/maya-io/2019/maya-io2019.SP2/bin/Render`
+
+5. Add the `submit_to_conductor.py` as a Job menu script. The path to the icon is `deadline_plugin/custom/scripts/General/conductor_logo.ico`
+![](../../image/deadline/script_menu_repo.jpg)
+
+## Usage
+### Prerequisites
+1. The Conductor upload daemon must be running
+2. The Conductor download daemon must be running
+3. A dependency sidecar exists for the given job
+
+###Sending Jobs to Conductor
+1. From the Deadline Monitor, select one or multiple jobs.
+2. From the context menu, select `Scripts->Send to Conductor`
```

Comment:
I read recently that mentions of UI elements the user should interact with, work better bold as opposed to in code tags:
From the context menu, select **Scripts->Send to Conductor**
Once you click **Submit**